src/matchers/toBeAccessible/index.spec.ts

```diff
@@ -109,4 +109,21 @@ test.describe.parallel('toBeAccessible', () => {
     expect(duration).toBeGreaterThan(1000)
     expect(duration).toBeLessThan(1500)
   })
+
+  test(`default report filename`, async ({ page }) => {
+    test.fail()
+    const content = await readFile('inaccessible.html')
+    await page.setContent(content)
+    await expect(page).toBeAccessible({ timeout: 2000 }).catch(() => { })
+    expect(attachmentExists('axe-report.html')).toBe(true)
+  });
+
+  test('should allow providing custom report filename', async ({ page }) => {
+    test.fail()
```

Comment:
```suggestion
```

Reply:
Ah yes, this is no longer required as I am catching the error instead. 👍